lib/view/ToDoPage.dart

```diff
@@ -57,7 +63,10 @@ class _ToDoPageState extends State<ToDoPage> {
             icon: const Icon(Icons.shopping_cart, size: 28),
             onPressed: () {
               store?.close();
-              Navigator.push(context, MaterialPageRoute(builder: (context) => const ShoppingMemoPage()));
+              Navigator.push(
+                  context,
+                  MaterialPageRoute(
+                      builder: (context) => const ShoppingMemoPage()));
```

Comment:
```suggestion
              Navigator.push(
                  context,
                  MaterialPageRoute(
                      builder: (context) => const ShoppingMemoPage()
                  )
              );
```
複数行に分ける場合は、改行を入れて欲しいです。